src/containers/forms/PlanForm/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,3225 @@
+  disabled={false}
+  id="activities-2-goalPriority"
+  name="activities[2].goalPriority"
+  onBlur={[Function]}
+  onChange={[Function]}
+  required={true}
+  value="medium-priority"
+>
+  <option
+    key="low-priority"
+    value="low-priority"
+  >
+    low-priority
+  </option>
+  <option
+    key="medium-priority"
+    value="medium-priority"
+  >
+    medium-priority
+  </option>
+  <option
+    key="high-priority"
+    value="high-priority"
+  >
+    high-priority
+  </option>
+</select>
+`;
+
+exports[`containers/forms/PlanForm renders activity fields correctly: FI activities[2].goalPriority label 1`] = `
+<Label
+  for="activities-2-goalPriority"
+  tag="label"
+  widths={
+    Array [
+      "xs",
+      "sm",
+      "md",
+      "lg",
+      "xl",
+    ]
+  }
+>
+  <label
+    className=""
+    htmlFor="activities-2-goalPriority"
+  >
+    Priority
+  </label>
+</Label>
+`;
+
+exports[`containers/forms/PlanForm renders activity fields correctly: FI activities[2].goalValue field 1`] = `
+<input
+  className="form-control"
+  disabled={false}
+  id="activities-2-goalValue"
+  name="activities[2].goalValue"
+  onBlur={[Function]}
+  onChange={[Function]}
+  required={true}
+  type="number"
+  value={100}
+/>
+`;
+
+exports[`containers/forms/PlanForm renders activity fields correctly: FI activities[2].goalValue label 1`] = `
+<Label
+  for="activities-2-goalValue"
+  tag="label"
+  widths={
+    Array [
+      "xs",
+      "sm",
+      "md",
+      "lg",
+      "xl",
+    ]
+  }
+>
+  <label
+    className=""
+    htmlFor="activities-2-goalValue"
+  >
+    Quantity
+  </label>
+</Label>
+`;
+
+exports[`containers/forms/PlanForm renders activity fields correctly: FI activities[2].timingPeriodEnd field 1`] = `
+<input
+  className="form-control"
+  disabled={false}
+  id="activities-2-timingPeriodEnd"
+  onBlur={[Function]}
+  onChange={[Function]}
+  onClick={[Function]}
+  onFocus={[Function]}
+  onKeyDown={[Function]}
+  readOnly={false}
+  required={true}
+  type="text"
+  value="2017-07-20"
+/>
+`;
+
+exports[`containers/forms/PlanForm renders activity fields correctly: FI activities[2].timingPeriodEnd label 1`] = `
+<Label
+  for="activities-2-timingPeriodEnd"
+  tag="label"
+  widths={
+    Array [
+      "xs",
+      "sm",
+      "md",
+      "lg",
+      "xl",
+    ]
+  }
+>
+  <label
+    className=""
+    htmlFor="activities-2-timingPeriodEnd"
+  >
+    End Date
+  </label>
+</Label>
+`;
+
+exports[`containers/forms/PlanForm renders activity fields correctly: FI activities[2].timingPeriodStart field 1`] = `
+<input
+  className="form-control"
+  disabled={false}
+  id="activities-2-timingPeriodStart"
+  onBlur={[Function]}
+  onChange={[Function]}
+  onClick={[Function]}
+  onFocus={[Function]}
+  onKeyDown={[Function]}
+  readOnly={false}
+  required={true}
+  type="text"
+  value="2017-07-13"
+/>
+`;
+
+exports[`containers/forms/PlanForm renders activity fields correctly: FI activities[2].timingPeriodStart label 1`] = `
```

Comment:
All the individual snapshots are now shorter and easier to read and reason about.